src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -345,6 +357,11 @@ public GitHubSCMSource(@CheckForNull String id, @CheckForNull String apiUri, @No
         }
     }
 
+    public String isURLRepo(){
```

Comment:
I'd tweak this method's name and signature:

```suggestion
    @Restricted(NoExternalUse.class)
    public boolean isConfiguredByURL() {
```